server/uno.go

```diff
@@ -120,6 +120,35 @@ func joinGame(game string, player *model.Player) (*model.Game, error) {
 	return gameData, nil
 }
 
+func addMessage(game string, username string, message model.Message) (*model.Game, error) { //*model.Player
```

Comment:
I'd love a unit test for this if possible.

Reply:
I can start working on this eventually but just in case I made an issue for it if someone else wanted to take it.

Reply:
I think considering the circumstances of encouraging team interaciton, that's fine with me.